.travis.yml

```diff
@@ -37,11 +37,11 @@ before_install:
 
 # Install packages with conda, then pip
 install:
-  - if [[ "$COVERAGE" == "true" ]]; then
-      export PIP_DEPS="$PIP_DEPS pytest-cov";
-    elif [[ "$STATIC" == "true" ]]; then
+  - if [[ "$STATIC" == "true" ]]; then
```

Comment:
I think I'd prefer the static case to be the else as it was before, both here and below... we run 9 tests total, and STATIC is only true in 1 of those tests, so it's always the exception.